airflow/contrib/operators/gcs_to_gcs.py

```diff
@@ -44,12 +44,17 @@ class GoogleCloudStorageToGoogleCloudStorageOperator(BaseOperator):
         storage bucket.
         If a wildcard is supplied in the source_object argument, this is the
         prefix that will be prepended to the final destination objects' paths.
+        Note that the source path's part before the wildcard will be removed;
```

Comment:
Can you add an example here?

Reply:
Will do, as mentioned below, in a new PR.